instance/jmsml_D_SIGINT_Space.xml

```diff
@@ -284,7 +284,7 @@
     </LegacySymbol>
     <LegacySymbol ID="SIGNAL_INTERCEPT_RADAR_UNKNOWN_MOD_SYM" Label="I*PPSRU-----***" EntityID="SIGNAL_INTERCEPT" EntityTypeID="RADAR" ModifierOneID="UNKNOWN_MOD">
       <LegacyFunctionCode Name="2525C">SRU---</LegacyFunctionCode>
-	    <LegacyEntity ID="SIGNAL_INTERCEPT_RADAR_UNKNOWN_MOD_LE" Label="I-P-SRU---.svg">
+	    <LegacyEntity ID="SIGNAL_INTERCEPT_RADAR_UNKNOWN_MOD_LE" Label="SIGINT.SPC.SIGINC.RAD.UNK" Graphic="I-P-SRU---.svg">
```

Comment:
So is there also a code change (like an exception handler added) associated with this fix?